surveyor/collector_statz.go

```diff
@@ -1184,3 +1219,52 @@ func unmarshalMsg(msg *nats.Msg, v any) error {
 
 	return json.Unmarshal(data, v)
 }
+
+type nameIDPair struct {
+	name string
+	id   uint64
+}
+
+func remapIDToIdx(pairs []nameIDPair, existingMapping map[string]map[uint64]int) map[string]map[uint64]int {
```

Comment:
I don't know if this is a good idea, because the route_id maps to something in the nats-server API:

Partial output from nats-server `/routez`:
```
{
  "server_name": "aws-apsoutheast2-natsc1-3",
  "num_routes": 8,
  "routes": [
    {
      "rid": 16,
      "remote_id": "...",
      "remote_name": "aws-apsoutheast2-natsc1-1",
```
If we remap these, then if someone wanted to debug something using a route_id from surveyor's labels then it wouldn't match up with what nats-server's API returns in the `rid` field.

Reply:
Yes but the high variance in route IDs is causing the number of time series for routes to grow almost unbounded, depending on the number of times the server is reloaded or routes reconnect

They're already not unique because RIDs start over when the server restarts

Plus even if they were unique, it's not recommended to use unique values as labels in order to reduce cardinality

Reply:
Good points, I've approved. Also noting that the routes are somewhat ephemeral, and tying a unique route_id to something that the server no longer has a record of is probably not that useful